### ISSUE TITLE
question: How to query the gitcoin subgraph for only accepted projects? 

### ISSUE THREAD

**Issue by ufkhan97** [gitcoinco/grants-stack#1031]
Hi all! 

@ccerv1 and I have been exploring the grants-round-mainnet subgraph for a couple of different use-cases such as building dune dashboards and creating hypercert metadata. 

We've queried for project information. However, we can't figure out how to filter out projects that haven't been accepted into the round. Specifically, the 'status' field for all projects is set to 'PENDING.'

As an example of what we're seeing, for the currently running Alpha OSS round the grant-explorer interface displays 89 projects but the query we're running returns 125. Our only solution so far has been to brute-force filter out the projects not appearing in the UI. 

Here is the query we're running:

```
  {
    rounds(where:{
        id: "0xd95a1969c41112cee9a2c931e849bcef36a16f4c"
    }) {
        id
        projects(first:300) {
        id
        project
        status
        payoutAddress
        metaPtr {
            protocol
            pointer
        }
        }
    }
    }
   ```

 [Repo for additional context](https://github.com/ufkhan97/Gitcoin-Helpers)
 
 To summarize, the question is: how do you filter out projects which aren't really in the round?

**Comment by gravityblast** [comment]
@ufkhan97 I guess we already solved it but let me know if you still need help with this
